appeals/web/src/styles/7-components/app-header.scss

```diff
@@ -40,6 +39,9 @@
 	  font-size: 1.5rem;
 	  width: 35%;
 	  text-align: center;
+	  @media (max-width: 48.0624em) {
```

Comment:
We should probably define this width value somewhere central as a variable. The existing instances seem to come from `$govuk-page-width`, but I'm not sure if there's a clean way we can import that from the govuk styles into our codebase